storybook/pages/ChangePasswordViewPage.qml

```diff
@@ -83,3 +83,5 @@ SplitView {
 }
 
 // category: Views
+
+// https://www.figma.com/file/d0G7m8X6ELjQlFOEKQpn1g/Profile-WIP?type=design&node-id=11-115317&mode=design&t=mBpxe2bJKzpseHGN-0
```

Comment:
@noeliaSD  was against figma links in code:
https://github.com/status-im/status-desktop/pull/14500#discussion_r1579468915

Reply:
That's in the code itself, this is a StoryBook page where it's actually needed, since we want to cross link the component with its design